master Update lib structure to acept 204 as success

gabe351 opened the pull request:
Signed-off-by: Gabriel Rosa <gabrielrosadn@gmail.com>

gabe351 committed:
master Update lib structure to acept 204 as success

-

Signed-off-by: Gabriel Rosa <gabrielrosadn@gmail.com>

rachidcalazans commented:
Is missing the tests. And you removed the option for exceptions. Probably the existents tests are failing.

gabe351 committed:
master Refactor tests structure

- add files for inner classses

Signed-off-by: Gabriel Rosa <gabrielrosadn@gmail.com>

gabe351 committed:
Update lib tests

- split tests in files per action
- extract common classes for specific files
- add new context for when have 204 status code

Signed-off-by: Gabriel Rosa <gabrielrosadn@gmail.com>

rachidcalazans commented:
@gabe351 Very nice changes man! Thanks!
